app/src/main/res/layout/fragment_search_food.xml

```diff
@@ -9,20 +9,14 @@
     <EditText
         android:id="@+id/search_text_view"
         android:layout_width="match_parent"
-        android:layout_height="56dp"
+        android:layout_height="@dimen/spaceXXLarge"
         android:layout_marginStart="@dimen/spaceLarge"
         android:layout_marginTop="@dimen/spaceMedium"
         android:layout_marginEnd="@dimen/spaceLarge"
```

Comment:
use layout_marginHorizntel="@dimen/spaceLarge" instead of marginStart and marginEnd